Sorts/TimSort.java

```diff
@@ -0,0 +1,177 @@
+package Sorts;
+
+import java.lang.Math;
+import java.util.Random;
+
+/**
+ * @author Hemanth Kotagiri (https://github.com/hemanth-kotagiri)
```

Comment:
```suggestion
 * @author [Hemanth Kotagiri](https://github.com/hemanth-kotagiri)
```

for links use markdown link syntax
```
[Text](link)
```